docs/3_setup_guide/Delft_Fiat/hydromt_fiat_gui.qmd

```diff
@@ -0,0 +1,229 @@
+---
+title:	Flood Adapt Model Builder - Graphical User Interface
+filters:
+  - lightbox
+lightbox: auto
+execute:
+  echo: false
+jupyter: python3
+---
+
+The Hydromt FIAT Graphical User Interface (GUI), also refered to as **Flood Adapt Model Builder (FIAT)**, facilitates the process of setting up the **Delft FIAT** model, thus making the **Delft FIAT** model creation accessible for users without modeling experience . 
+
+# Setup Guide
+## Installation
+To run the Floodadapt model builder you need to install two tools:
+
+  1. **HydroMT FIAT**
+  2. **DelftDashboard**
+
+#### HydroMT FIAT
+HydroMT-FIAT is available from pypi and will be added to conda-forge (in progress).
+
+To install hydromt_fiat for usage, do:
+
+  > pip install hydromt_fiat
+
+#### DelftDashboard
+For the Graphical User Interface you additionally need to set up **DelftDashboard**. You can find the installation guide [here](https://github.com/Deltares-research/DelftDashboard/blob/FloodAdapt_ModelBuilders/README.md).
+
+# User Guide
+After you open the GUI you must select a model output directory before you start.
+
+*GUI instructions*
+  
+  1. Go to **File** >> **Select Working Directory**.
+  2. Select an output folder.
+
+Now you are ready to set up the model.
+
+After creating your model you should end up with a folder structure as describted in the section [Data Input - Folder Structure](data_input.qmd).
+
+## Create Model Boundary
+
+There are multiple options on how to create the model boundaries:
+
+  * Draw polygon/ draw box  
+  * SFINCS model domain 
+  * Upload file (polygon vector file e.g. *.gpkg*, *.geojson*)
+
+![Workflow to create model boundaries.](../../_static/figures/Draw_boundaries.svg){#fig-boundaries}
+
+*GUI instructions*
```

Comment:
I wouldn't say "GUI instructions". Maybe something like 'To select a model boundary, you have four options:" and then put the list of options. Then say "when you are finished, click "Generate Boundary".